client/components/public/imports/SponsorRow.jsx

```diff
@@ -27,7 +27,7 @@ class SponsorRowInner extends Component {
   }
 }
 
-SponsorRowInner.PropTypes = {
+SponsorRowInner.propTypes = {
```

Comment:
This seems like the only place where `propTypes` has been changed to be lowercase. Why?